index.bs

```diff
@@ -264,19 +280,19 @@ A file containing items compliant with this profile is expected to list the foll
 
 A file containing a 'pict' track compliant with this profile is expected to list the following brands, in any order, in the [=compatible_brands=] of the [=FileTypeBox=]:
 
-  <code>avis, msf1, miaf, MA1B</code>
+  <code>avis, avif, msf1, miaf, MA1B</code>
```

Comment:
[Just a comment, not a request for change] Listing both "avis" and "avif" increases the image container overhead. Container overhead is a known problem with AVIF. It seems bad to make the problem worse.